This patch fixes declaration of onStatement function (taken from CVS)

--- php-pecl-apd-1.0.1/apd-1.0.1/php_apd.c~	2004-09-28 05:25:59.000000000 +0200
+++ php-pecl-apd-1.0.1/apd-1.0.1/php_apd.c	2004-11-01 10:59:46.123975488 +0100
@@ -62,7 +62,7 @@
 #endif
 ZEND_DLEXPORT void (*old_execute)(zend_op_array *op_array TSRMLS_DC);
 
-ZEND_DLEXPORT void onStatement(zend_op_array *op_array TSRMLS_DC);
+ZEND_DLEXPORT void onStatement(zend_op_array *op_array);
 ZEND_DECLARE_MODULE_GLOBALS(apd);
 
 /* This comes from php install tree. */
